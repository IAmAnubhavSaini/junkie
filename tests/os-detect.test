#!/bin/sh

NAME="OS detection"
# windows.7or8.pcap was obtained from microsoft.com, which HTTP server pretends to be ISS.
# But we do not detect it as such (nor at all). Would it be possible for microsoft to use another OS?
PCAPS="pcap/os-detection/linux.2.6.32.pcap pcap/os-detection/windows.7.pcap pcap/os-detection/windows.vista.pcap pcap/os-detection/openbsd.pcap"
PLUGIN="../plugins/os-detect/.libs/os-detect.so"

filter() {
	cat -
}
OUTPUT="os-detect.out"

. "$srcdir/dump_test"
