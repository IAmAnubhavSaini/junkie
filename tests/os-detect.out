-- pcap/os-detection/linux.2.6.32.pcap --
192.168.10.9: g:unix:Linux:2.2.x-3.x (no timestamps)
-- pcap/os-detection/windows.7.pcap --
192.168.20.3: g:win:Windows:NT kernel
-- pcap/os-detection/windows.vista.pcap --
10.1.0.56: g:win:Windows:NT kernel
-- pcap/os-detection/openbsd.pcap --
192.168.10.254: s:unix:OpenBSD:5.x
